docs/security/host-certs.md

```diff
@@ -137,252 +114,79 @@ for more details.
-
-        :::console
-        root@host # chown -R %RED%<USER>%ENDCOLOR%:%RED%<USER>%ENDCOLOR% /etc/grid-security/%RED%<SERVICE>%ENDCOLOR%/
-
-    For example, both the Apache and Tomcat services are run by the `tomcat` user:
-
-        :::console
-        root@host # chown -R tomcat:tomcat /etc/grid-security/http/
-
-Requesting Host/Service Certificate Using OIM
-----------------------------------------------
-
-If you do not have Grid Admin privileges, please use OIM to request any host or
-service certificates that you may need.  The OSG PKI Certificate Request &
-Management System can be found at:
-<https://oim.opensciencegrid.org/oim/certificate>. Alternatively, you can request
-Grid Admin privileges [here](https://oim.opensciencegrid.org/oim/gridadmin) after obtaining
-your [user certificate](user-certs).
+In the case where your host is only running a single service that requires a service certificate, we recommend using
+your host certificate as your service certificate.
+Ensure that the ownership of the host certificate and key are appropriate for the service you are running.
 
+If you are running multiple services that require host certificates, we recommend requesting a certificate whose
+CommonName is `<service>-hostname` and has the hostname in the list of subject alternative names.
 
 Frequently Asked Questions
 ---------------------------
 
 ### Can I use any host to request a certificate for a different host? 
 
-YES, you can use any host to create a certificate request as long as the hostname for the certificate is a fully qualified domain name.
-
-### May I reuse my host certificate as a service certificate? 
+YES, you can use any host to create a certificate signing request as long as the hostname for the certificate is a fully
+qualified domain name.
 
-NO! For security reasons, please do not use clones of your host certificate for additional certificates even though it's technically possible.
+### How do I renew a host certificate? 
 
-### How do I renew a host/service certificate? 
+For Let's Encrypt certificates, see [this section](#renewing-lets-encrypt-host-certificates)
 
-There is no separate procedure. Simply ask for a new certificate the same way you asked for it the previous time.
-
-### I get a "GSS authentication failure" when users try to authenticate with my site? 
-
-You likely used an **alias** for the host instead of the fully qualified domain
-name when you generated the certificate request. This can cause the GSS
-authentication failures similar to the following when a user tries to
-authenticate to the host after your certificate is installed:
-
-``` console
-GSS authentication failure 
-GSS Major Status: General failure 
-GSS Minor Status Error Chain: 
-accept_sec_context.c:gss_accept_sec_context:403: 
-Error during delegation: Delegation protocol violation 
-Failure: GSS failed Major:000d0000 Minor:00000001 Token:00000000 
-```
+There is no separate renewal procedure.
```

Comment:
Should be more like "For other certificates, there is no separate renewal procedure."